Sources/Client/Model/Channel/Channel.swift

```diff
@@ -117,6 +117,9 @@ public final class Channel: Codable {
     
     private var subscriptionBag = SubscriptionBag()
     
+    var isCurrentUserTyping = false
```

Comment:
This needs to be `atomic`

Reply:
It's should be always called from the main thread.

Reply:
I don't see a reason why this function must be called from the main thread tbh.